package.json

```diff
@@ -115,13 +119,15 @@
     "gulp-less": "^5.0.0",
     "gulp-postcss": "^9.0.0",
     "hardhat": "^2.6.7",
+    "history": "^5.2.0",
     "jest": "26.6.0",
     "jest-puppeteer": "^6.0.0",
     "less-plugin-npm-import": "^2.1.0",
     "msw": "^0.36.3",
     "pptr-testing-library": "^0.6.5",
     "prettier": "^2.0.5",
     "puppeteer": "^11.0.0",
+    "shelljs": "^0.8.5",
```

Comment:
what are `history` & `shelljs` needed for?

Reply:
shelljs is used by the e2e tests: https://github.com/OlympusDAO/olympus-frontend/blob/develop/tests/e2e/testHelpers.ts#L6

Reply:
history is used by testUtils: https://github.com/OlympusDAO/olympus-frontend/blob/develop/src/testUtils.js#L10